src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/LanguageServiceHost.cs

```diff
@@ -81,7 +78,7 @@ internal partial class LanguageServiceHost : OnceInitializedOnceDisposedAsync, I
 
         [ProjectAutoLoad(completeBy: ProjectLoadCheckpoint.ProjectFactoryCompleted)]
         [AppliesTo(ProjectCapability.DotNetLanguageService)]
-        private Task OnProjectFactoryCompletedAsync()
+        public Task OnProjectFactoryCompletedAsync()
```

Comment:
Why is this public now?

Reply:
The analyzer doesn't understand MEF exports - public makes it quiet.